packages/app/templates/ui-extensions/projects/ui_extension/shopify.ui.extension.toml.liquid

```diff
@@ -3,6 +3,7 @@ name = "{{ name }}"
 [capabilities]
 network_access = true
 block_progress = true
+api_access = true
```

Comment:
Super nitpicky thing: I think this is going to be the most widely-used capability, so can we put it above the other two? It is also a little shorter and my brain feels better when lines go from short to long 😆